Common/Securities/SecurityCache.cs

```diff
@@ -37,7 +37,7 @@ public class SecurityCache
         /// <summary>
         /// Gets the most recent price submitted to this cache
         /// </summary>
-        public decimal Price { get; private set; }
+        public decimal Price { get; set; }
```

Comment:
SetMarketPrice is updating volatility models so would cause different data in backtest/live; but might be good to have a SetRealTimePrice() setter method rather than setting the property directly